### PR TITLE
Storage for the web

### LINKED ISSUE #2634: content: Storage on the web [May 20 2020]
hard deadline: no

author: @petele 

draft: https://docs.google.com/document/d/1f-iDXQwrBJWsfAkc8n1w8EEfu--IxjDTRqSBLBkqjkg/edit#

related content: 

* https://developers.google.com/web/fundamentals/instant-and-offline/web-storage/offline-for-pwa
* https://www.html5rocks.com/en/tutorials/offline/quota-research/

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- Adds new article on storage for the web

Fixes #2634

### CODE REVIEW

**Comment by petele** [comment]
@jpmedley - integrated your feedback from the Google doc and did the markdown conversion.

**Comment by kaycebasques** [comment]
FYI removing the labels because we have an issue tracking this already and the work is now showing up twice in the ZenHub tracker

File: src/site/content/en/blog/storage-for-the-web/index.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+the actual disk size. This helps to reduce the ability to determine the size
+of stored cross origin resources.
+{% endAside %}
+
+### Inspect
+
+During development, you can use your browser's DevTools to inspect the
+different storage types, and easily clear all stored data.
+
+![Storage test tool](./storage-test-tool.png)
```
Comment (kaycebasques) [inline src/site/content/en/blog/storage-for-the-web/index.md:153]:
```suggestion
<img class="w-screenshot w-screenshot-filled" alt="Storage test tool." src="storage-test-tool.png">
```

**Comment by kaycebasques** [comment]
@petele you don't want this to live under web.dev/pwa or web.dev/reliable? It seems like very authoritative and canonical guidance and is therefore well-suited to one of those locations. It looks like there isn't a great place to put this in those two locations, but I can fix that.

I've approved the PR so you can merge it now. I just wanted to check about this before merging.

**Comment by petele** [comment]
> you don't want this to live under web.dev/pwa or web.dev/reliable?

@kaycebasques - I considered /pwa, though not /reliable. I feel it's widely useful, and doesn't really fit either category nicely. For example, the IDB stuff isn't just about reliable. Open to suggestions about moving it into one of those categories. Will file an issue to discuss.

**Pull request merged**